<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>881</width>
      <height>591</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 247, 199);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>gestion stock</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>231</width>
        <height>301</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajouter au stock</string>
      </property>
      <widget class="QPushButton" name="pushButton_ajouterstock">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton {
	
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
background-color: rgb(181, 216, 131);
border:2px solid #000;
}
QPushButton {

     color: #fff !important;
text-transform: uppercase;
text-decoration: none;
background: #b5d883;
padding: 20px;
border-radius: 5px;
display: inline-block;
border: none;
transition: all 0.4s ease 0s;
 }
 QPushButton:hover {
background-image: url(:/image/Nouveau dossier/check-mark.png);
     background: #f68623;
letter-spacing: 1px;
-webkit-box-shadow: 0px 5px 40px -10px rgba(0,0,0,0.55);
-moz-box-shadow: 0px 5px 40px -10px rgba(0,0,0,0.55);
box-shadow: 5px 40px -10px rgba(0,0,0,0.55);
transition: all 0.4s ease 0s;
 }</string>
       </property>
       <property name="text">
        <string>ajouter</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>id_stock</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_idstock">
       <property name="geometry">
        <rect>
         <x>92</x>
         <y>30</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(131, 170, 135);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>lib_prod</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit_date">
       <property name="geometry">
        <rect>
         <x>88</x>
         <y>170</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(131, 170, 135);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>170</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>date</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_qte">
       <property name="geometry">
        <rect>
         <x>92</x>
         <y>130</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(131, 170, 135);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Qte</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>description</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_libprod">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>80</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(131, 170, 135);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_desc">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>220</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(131, 170, 135);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_ajouteract_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>260</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton {
	
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
background-color: rgb(181, 216, 131);
border:2px solid #000;
}
QPushButton {

     color: #fff !important;
text-transform: uppercase;
text-decoration: none;
background: #b5d883;
padding: 20px;
border-radius: 5px;
display: inline-block;
border: none;
transition: all 0.4s ease 0s;
 }
 QPushButton:hover {
background-image: url(:/image/Nouveau dossier/check-mark.png);
     background: #f68623;
letter-spacing: 1px;
-webkit-box-shadow: 0px 5px 40px -10px rgba(0,0,0,0.55);
-moz-box-shadow: 0px 5px 40px -10px rgba(0,0,0,0.55);
box-shadow: 5px 40px -10px rgba(0,0,0,0.55);
transition: all 0.4s ease 0s;
 }</string>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>581</width>
        <height>291</height>
       </rect>
      </property>
      <property name="title">
       <string>Afficher le stock</string>
      </property>
      <widget class="QTableView" name="tableView_stock">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>571</width>
         <height>261</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>231</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Supprimer activit√©</string>
      </property>
      <widget class="QComboBox" name="comboBox_3_idact">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>73</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>id_act</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_suppract_3">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>160</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton {
	
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
background-color: rgb(181, 216, 131);
border:2px solid #000;
}
QPushButton {

     color: #fff !important;
text-transform: uppercase;
text-decoration: none;
background: #b5d883;
padding: 20px;
border-radius: 5px;
display: inline-block;
border: none;
transition: all 0.4s ease 0s;
 }
 QPushButton:hover {
background-image: url(:/image/Nouveau dossier/check-mark.png);
     background: #f68623;
letter-spacing: 1px;
-webkit-box-shadow: 0px 5px 40px -10px rgba(0,0,0,0.55);
-moz-box-shadow: 0px 5px 40px -10px rgba(0,0,0,0.55);
box-shadow: 5px 40px -10px rgba(0,0,0,0.55);
transition: all 0.4s ease 0s;
 }</string>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_9">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>320</y>
        <width>251</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>rechercher</string>
      </property>
      <widget class="QLineEdit" name="lineEdit_rech">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>60</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(131, 170, 135);</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_crit">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>20</y>
         <width>73</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>idstock</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>lib_prod</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>datee</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>critere</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_11">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>440</y>
        <width>251</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string>generer fichier pdf</string>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>40</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton {
	
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
background-color: rgb(181, 216, 131);
border:2px solid #000;
}
QPushButton {

     color: #fff !important;
text-transform: uppercase;
text-decoration: none;
background: #b5d883;
padding: 20px;
border-radius: 5px;
display: inline-block;
border: none;
transition: all 0.4s ease 0s;
 }
 QPushButton:hover {
background-image: url(:/image/Nouveau dossier/check-mark.png);
     background: #f68623;
letter-spacing: 1px;
-webkit-box-shadow: 0px 5px 40px -10px rgba(0,0,0,0.55);
-moz-box-shadow: 0px 5px 40px -10px rgba(0,0,0,0.55);
box-shadow: 5px 40px -10px rgba(0,0,0,0.55);
transition: all 0.4s ease 0s;
 }</string>
       </property>
       <property name="text">
        <string>pdf</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>500</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
	
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
background-color: rgb(181, 216, 131);
border:2px solid #000;
}
QPushButton {

     color: #fff !important;
text-transform: uppercase;
text-decoration: none;
background: #b5d883;
padding: 20px;
border-radius: 5px;
display: inline-block;
border: none;
transition: all 0.4s ease 0s;
 }
 QPushButton:hover {
background-image: url(:/image/Nouveau dossier/check-mark.png);
     background: #f68623;
letter-spacing: 1px;
-webkit-box-shadow: 0px 5px 40px -10px rgba(0,0,0,0.55);
-moz-box-shadow: 0px 5px 40px -10px rgba(0,0,0,0.55);
box-shadow: 5px 40px -10px rgba(0,0,0,0.55);
transition: all 0.4s ease 0s;
 }</string>
      </property>
      <property name="text">
       <string>stats</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_tri">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>370</y>
        <width>75</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
	
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
background-color: rgb(181, 216, 131);
border:2px solid #000;
}
QPushButton {

     color: #fff !important;
text-transform: uppercase;
text-decoration: none;
background: #b5d883;
padding: 20px;
border-radius: 5px;
display: inline-block;
border: none;
transition: all 0.4s ease 0s;
 }
 QPushButton:hover {
background-image: url(:/image/Nouveau dossier/check-mark.png);
     background: #f68623;
letter-spacing: 1px;
-webkit-box-shadow: 0px 5px 40px -10px rgba(0,0,0,0.55);
-moz-box-shadow: 0px 5px 40px -10px rgba(0,0,0,0.55);
box-shadow: 5px 40px -10px rgba(0,0,0,0.55);
transition: all 0.4s ease 0s;
 }</string>
      </property>
      <property name="text">
       <string>tri</string>
      </property>
     </widget>
     <widget class="QComboBox" name="colone_tri">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>370</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>idstock</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>lib_prod</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>datee</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="ordre_tri">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>370</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>ASC</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DESC</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="Editdispolibprod">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>430</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(131, 170, 135);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>430</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
	
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
background-color: rgb(181, 216, 131);
border:2px solid #000;
}
QPushButton {

     color: #fff !important;
text-transform: uppercase;
text-decoration: none;
background: #b5d883;
padding: 20px;
border-radius: 5px;
display: inline-block;
border: none;
transition: all 0.4s ease 0s;
 }
 QPushButton:hover {
background-image: url(:/image/Nouveau dossier/check-mark.png);
     background: #f68623;
letter-spacing: 1px;
-webkit-box-shadow: 0px 5px 40px -10px rgba(0,0,0,0.55);
-moz-box-shadow: 0px 5px 40px -10px rgba(0,0,0,0.55);
box-shadow: 5px 40px -10px rgba(0,0,0,0.55);
transition: all 0.4s ease 0s;
 }</string>
      </property>
      <property name="text">
       <string>dispo</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>tab2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>897</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
